masschallenge/semantic-ui-theme
# [AC 7408] Create new component: Office Hours Time Cards

## [AC 7408](https://masschallenge.atlassian.net/browse/AC-7408)

**Changes Introduced:**
- Add office hours time card styles

**Testing:**
- If you haven't already, setup [semantic-ui-theme](https://github.com/masschallenge/semantic-ui-theme/blob/development/README.md) on your development environment (you may have to downgrade your node version to 10.*.  command: `n 10.16.0`)
- run `gulp build`
- run `gulp watch`
- `yarn link`
- `cd ../accelerate && yarn link "semantic-ui-theme"`

Continue testing on [sibling PR](https://github.com/masschallenge/accelerate/pull/2563)

This PR should be merged simultaneously with [sibling PR](https://github.com/masschallenge/accelerate/pull/2563)
